tests/Unit/IO/Test_StellarCollapseEos.cpp

```diff
@@ -0,0 +1,353 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <string>
+#include <vector>
+
+#include "DataStructures/BoostMultiArray.hpp" // IWYU pragma: keep
+#include "IO/H5/AccessType.hpp"
+#include "IO/H5/File.hpp"
+#include "IO/H5/Header.hpp"  // IWYU pragma: keep
+#include "IO/H5/StellarCollapseEos.hpp"
+#include "Informer/InfoFromBuild.hpp"
+#include "tests/Unit/TestingFramework.hpp"
+
+// IWYU pragma: no_include <boost/iterator/iterator_facade.hpp>
+// IWYU pragma: no_include <boost/multi_array.hpp>
+// IWYU pragma: no_include <boost/multi_array/base.hpp>
+// IWYU pragma: no_include <boost/multi_array/extent_gen.hpp>
+// IWYU pragma: no_include <boost/multi_array/multi_array_ref.hpp>
+// IWYU pragma: no_include <boost/multi_array/subarray.hpp>
+
+namespace {
+
+// For 1D arrays the numbers (in, for example, logrho_expected) are the first
```

Comment:
What do you mean by "the numbers"? Are you saying you are only checking the first two numbers in the .h5 file? Or are you describing the contents of the .h5 file?

Reply:
I updated the comment to make it clear that these are numbers in the HDF5 file